charts/newrelic-infrastructure-beta/templates/control-plane/configmap.yaml

```
@@ -0,0 +1,16 @@
+{{- if .Values.controlPlane.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}-control-plane
+data:
+  nri-kubernetes-config.yml: |
+    {{- .Values.common.config | toYaml | nindent 4 }}
+    controlPlane:
+      enabled: true
+    {{- if .Values.controlPlane.config -}}
+    {{- .Values.controlPlane.config | toYaml | nindent 6 }}
```

This will not get the `config` key.
```suggestion
    config:
      {{- .Values.controlPlane.config | toYaml | nindent 6 }}
```

---

i think that `config:` is not needed actually 
https://github.com/newrelic/nri-kubernetes/blob/develop/internal/config/config.go#L47